src/Mount.cpp

```diff
@@ -107,19 +67,17 @@ const float siderealDegreesInHour = 14.95904348958;
 // CTOR
 //
 /////////////////////////////////
-Mount::Mount(float stepsPerRADegree, float stepsPerDECDegree, LcdMenu* lcdMenu) {
-   _instance = this;
-   
-  #if RA_DRIVER_TYPE != DRIVER_TYPE_ULN2003
-  _stepsPerRADegree = stepsPerRADegree * SET_MICROSTEPPING; // hier
-  #else
-  _stepsPerRADegree = stepsPerRADegree;
-  #endif
-  #if DEC_DRIVER_TYPE != DRIVER_TYPE_ULN2003
-  _stepsPerDECDegree = stepsPerDECDegree * DEC_SLEW_MICROSTEPPING;
-  #else
-  _stepsPerDECDegree = stepsPerDECDegree;
+Mount::Mount(LcdMenu* lcdMenu) :
+  _stepsPerRADegree(RA_STEPS_PER_DEGREE),   // u-steps per degree when slewing
+  _stepsPerDECDegree(DEC_STEPS_PER_DEGREE)  // u-steps per degree when slewing
+  #if AZIMUTH_ALTITUDE_MOTORS == 1
+    , _stepsPerAZDegree(AZIMUTH_STEPS_PER_REV / 360),
+    _stepsPerALTDegree(ALTITUDE_STEPS_PER_REV / 360),
+    _azAltWasRunning(false)
   #endif
+{
+   _instance = this;
```

Comment:
if i see it correctly, we don't use this singleton anywhere ... could be deleted

Reply:
Agreed. I think this is legacy of very old interrupt handling on ATmega.

Reply:
yeah probably

Reply:
I think the Meade processor used to pull this... but we changed that and injected it instead. So, yeah, remove.

Reply:
Done.